### ISSUE TITLE
Unable to create or open a dataset in Power BI Service with an Ampersand in the name

### ISSUE THREAD

**Issue by edhans** [TabularEditor/TabularEditor3#872]
**Describe the issue**
A clear and concise description of what the issue is.

** Which version are you currently using? **
3.5.1

**To Reproduce**
Power BI Service - Premium Capacity

Connect to the service and name your dataset anything with an & character. It will return an error that the database name cannot contain special characters. 

**Expected behavior**
The ampersand is not a restricted character. We already have datasets with that in the name, and I work with BIM files connected to the workspace with the & in it. I just cannot start a new one. I have to remove the ampersand, connect, rename in the service, export my BIM, and reconnect.

**Screenshots**
![image](https://user-images.githubusercontent.com/45298358/229840482-a828e57b-eb4a-42f1-90fe-0cc6ca043803.png)

**Desktop (please complete the following information):**
 - Windows version:  10

**Additional context**

**Comment by edhans** [comment]
This is worse than I thought. I cannot even open databases with an & in the name now. the OK button is grayed out for any dataset with & in it.


I am going to have to revert to 3.5.0.
![DBwithAmpersand](https://user-images.githubusercontent.com/45298358/229855287-8f050283-f2ab-42f7-b8e8-cd55021f2391.gif)

**Comment by otykier** [comment]
Yes, this is a known issue in 3.5.1 whenever the database name contains special characters. A fix will be available in the next update.

As a workaround, if you do not want to downgrade, it is possible to launch TE3 through a command line, with the XMLA server name and dataset name as parameters:

```shell
tabulareditor3.exe "powerbi://api.powerbi.com/v1.0/myorg/MyWorkspace" "GO&SC"
```

**Comment by DBojsen** [comment]
Hi @edhans 

This is now fixed in [3.6.0](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.6.0).
Thanks for reporting.

BR David - TE3 Support
